docs/docs/technology/introduction.md

```diff
@@ -87,15 +87,22 @@ flowchart LR
 
 #### **Main Translation Steps:**
 
-1. **text-to-SignWriting Translation**: Spoken language text is translated into SignWriting sequences using machine translation (it sees a lot of data, and extrapolates from it).
+1. **text-to-SignWriting Translation**: Spoken language text is translated into SignWriting sequences using machine translation (it sees a lot of data, and extrapolates from it). Timothy Comment: What is the conntext for this? are you explaining something to the reader here? 
 
-2. **SignWriting-to-pose conversion**: The written signs are converted into a fluent pose sequence, illustrating how the signs would be physically performed by a signer.
+2. **SignWriting-to-pose conversion**: The written signs are converted into a fluent pose sequence, illustrating how the signs would be physically performed by a signer. Timothy Comment: would this not be more accurate: The Signwriting sequences are converted into fluent pose sequence that illustrates how the sign would be performed by a signer. 
```

Comment:
you could just make the edit you want. then, if i have rejects, i'll tell you.
- `SignWriting` is capitalized as such.
- "a" fluent pose sequence
- removing "physically" is good imo